src/rail/cli/rail_pipe/pipe_scripts.py

```diff
@@ -16,6 +16,24 @@
 from rail.cli.rail_pipe.pipe_options import RunMode
 
 
+S3DF_SLURM_OPTIONS: list[str] = [
+    "-p",
+    "milano",
+    "--account",
+    "rubin:commissioning@milano",
+    "--mem",
+    "16448",
+    "--parsable",
+]
+NERSC_SLURM_OPTIONS: list[str] = [
+]
+
+SLURM_OPTIONS = {
+    "s3df":S3DF_SLURM_OPTIONS,
+    "nersc":NERSC_SLURM_OPTIONS,
```

Comment:
should we use `perlmutter` instead of `nersc` here?

Reply:
Do you feel strongly?   It is longer to type.  :)

Reply:
`s3df` is longer to type than `slac`, too :)

but no, not strongly